### ISSUE TITLE
All requests error out

### ISSUE THREAD

**Issue by asood92** [v0idp/Mellow#47]
Movie requests error out after clicking the download icon to actually send the request to ombi. TV requests don't even search now.

Reset the container and set it up from scratch hoping it'd fix it, no luck.

First error is from a movie request, the one after is after trying to search a tv show to add.
Container logs:
```

2020-01-14T01:22:19: PM2 log: Launching in no daemon mode
2020-01-14T01:22:19: PM2 log: App [index:0] starting in -fork mode-
2020-01-14T01:22:19: PM2 log: App [index:0] online
There is no bot token provided. Please check your configuration!
WebServer ready and listening on port 5060. Webroot for static files set to /usr/src/app/src/views/
No BotClient is running to restart. Starting a new BotClient...
BotClient ready and logged in as PlexBot (##################). Prefix set to !plex. Use !plexhelp to view the commands list!
{
error: Error: Invalid protocol: 192.168.1.20:
at Request.init (/usr/src/app/node_modules/request/request.js:458:31)
at new Request (/usr/src/app/node_modules/request/request.js:127:8)
at request (/usr/src/app/node_modules/request/index.js:53:10)
at Function.post (/usr/src/app/node_modules/request/index.js:61:12)
at /usr/src/app/src/util.js:33:17
at Promise._execute (/usr/src/app/node_modules/bluebird/js/release/debuggability.js:384:9)
at Promise._resolveFromExecutor (/usr/src/app/node_modules/bluebird/js/release/promise.js:518:18)
at new Promise (/usr/src/app/node_modules/bluebird/js/release/promise.js:103:10)
at post (/usr/src/app/src/util.js:32:12)
at /usr/src/app/src/commands/ombi/movie.js:89:5
at processTicksAndRejections (internal/process/task_queues.js:94:5),
body: undefined
}
{
error: Error: Invalid protocol: 192.168.1.20:
at Request.init (/usr/src/app/node_modules/request/request.js:458:31)
at new Request (/usr/src/app/node_modules/request/request.js:127:8)
at request (/usr/src/app/node_modules/request/index.js:53:10)
at Function.get (/usr/src/app/node_modules/request/index.js:61:12)
at /usr/src/app/src/util.js:24:17
at Promise._execute (/usr/src/app/node_modules/bluebird/js/release/debuggability.js:384:9)
at Promise._resolveFromExecutor (/usr/src/app/node_modules/bluebird/js/release/promise.js:518:18)
at new Promise (/usr/src/app/node_modules/bluebird/js/release/promise.js:103:10)
at get (/usr/src/app/src/util.js:23:12)
at searchTVCommand.run (/usr/src/app/src/commands/ombi/tv.js:154:4)
at async CommandoMessage.run (/usr/src/app/node_modules/discord.js-commando/src/extensions/message.js:235:20)
at async CommandDispatcher.handleMessage (/usr/src/app/node_modules/discord.js-commando/src/dispatcher.js:143:19),
body: undefined
}

```

**Comment by v0idp** [comment]
Did you add a port to your ombi settings in web configuration?

**Comment by Migz93** [comment]
Maybe also worth putting http:// in front of your IP address

**Comment by asood92** [comment]
Yes, default port for ombi is 3579, its what I've been using and is mapped properly. It's set in the mellow webui. 

Added http:// in front of the IP, changed the error to:

{
error: null,

body: '404 Site 192.168.1.20 is not served on this interface\n'
}
{
error: null,

body: '404 Site 192.168.1.20 is not served on this interface\n'
}

**Comment by v0idp** [comment]
Can you even search movies or tv shows at all ? Request only includes pressing the download reaction.

**Comment by asood92** [comment]
With the http:// removed, I can search for movies no problem. !plex movie movie.name works fine, pops up the list of movies from The Movie DB, selecting the title works fine, it populates the movie info blurb and sends the "If you want to request this movie please click on the :arrow_down: reaction." message properly. Clicking the reaction is where it errors out, returning "There was an error in your request."

Searching a TV show immediately errors out. goes no further than !plex tv show.name -> "There was an error in your request."

**Comment by v0idp** [comment]
can you post your ombi settings

**Comment by asood92** [comment]
Host - 192.168.1.20
Port - 3579
API Key - #############

Docker mappings for ombi / mellow
<img width="1131" alt="Screenshot 2020-01-14 09 41 45" src="https://user-images.githubusercontent.com/392075/72353709-7c8e4a80-36b2-11ea-8603-2555e0c3fec4.png">

**Comment by v0idp** [comment]
fixed in branch develop in latest commit a605880
